deployment/pypi/setup.py

```diff
@@ -15,6 +17,35 @@
 else:
     raise NotImplementedError('current platform {} not supported'.format(os_type))
 
+class AutoCompletion(install):
```

Comment:
better to remove installation, and update doc to explain how to enable bash completion, or output message after nni installation to tell users to enable bash completion

Reply:
Installation removed. An [tutorial documentation](https://github.com/microsoft/nni/blob/0153b3f6eff4cce807857bcee30c872fab63caf2/docs/en_US/CommunitySharings/AutoCompletion.md) has been added.